src/Test.FunctionalTests.Sarif/Test.FunctionalTests.Sarif.csproj

```diff
@@ -3551,6 +3551,9 @@
     <None Update="v2\ConverterTestData\FxCop\SarifSdkTest.xml.sarif">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="v2\ConverterTestData\PoliCheck\PoliCheck5.9.1_Sample001.xml">
```

Comment:
i could not figure out how the other test files (not *.sarif) are magically appearing in build output subfolder. do tell!